index.js

```diff
@@ -5,14 +5,25 @@ module.exports = {
   name: 'ember-welcome-page',
 
   included: function(app) {
-    this._super.included(app);
+    this._super.included.apply(this, arguments);
+
     this._welcomeConfig = app.options['ember-welcome-page'] || {};
 
     if (this._isDisabled()) { return; }
 
     app.import('vendor/welcome-page.css');
   },
 
+  config: function() {
+
+    const project = this.app && this.app.project;
```

Comment:
Can you swap this to:

```suggestion
    const project = this.project;
```

Reply:
Resolved by https://github.com/ember-cli/ember-welcome-page/pull/124/commits/5926a559312df28ab44167bc13ae27b882149e31